iD.validations.crossing_ways test fails — flags sidewalk crossing service road

@erictheise noticed a while ago that this test [started failing](https://github.com/OpenHistoricalMap/iD/actions/runs/8635681723/job/23673970782):

> Chrome Headless 123.0.6312.105 (Linux x86_64) iD.validations.crossing_ways flags sidewalk crossing service road FAILED
>	AssertionError: expected [ validationIssueFix{ …(8) }, …(5) ] to have a length of 5 but got 6
>	    at Context.<anonymous> (test/spec/validations/crossing_ways.js:273:38)

https://github.com/OpenHistoricalMap/iD/blob/87d6795e91606953fb1c9a5bd8caba1ad4bc1ed4/test/spec/validations/crossing_ways.js#L273

--- a/test/spec/validations/crossing_ways.js
+++ b/test/spec/validations/crossing_ways.js
@@ -270,7 +270,7 @@ describe('iD.validations.crossing_ways', function () {
         verifySingleCrossingIssue(issues, {});
         context.enter(iD.modeSelect(context, ['w-1']));
         const dynamicFixes = issues[0].dynamicFixes(context);
-        expect(dynamicFixes).to.have.lengthOf(5);
+        expect(dynamicFixes).to.have.lengthOf(6);
         expect(dynamicFixes[0]._connectionTags).to.eql({});
         expect(dynamicFixes[1]._connectionTags).to.eql({ highway: 'crossing' });
     });

